plugin-3.0/src/main/java/org/jvnet/mjiip/v_3_0/LoggingErrorReceiver.java

```diff
@@ -0,0 +1,64 @@
+/**
+ *
```

Comment:
No license header?

Reply:
This was copied from the `v_2_3` package pretty much unmodified.